lib/cryptoexchange/exchanges/c2cx/services/pairs.rb

```diff
@@ -0,0 +1,29 @@
+module Cryptoexchange::Exchanges
+  module C2cx
+    module Services
+      class Pairs < Cryptoexchange::Services::Pairs
+        
+        PAIRS_URL = "#{Cryptoexchange::Exchanges::C2cx::Market::API_URL}/ExchangePolicy"
+
+        def fetch
+          response = super
+          adapt(response)
+        end
+
+        def adapt(response)
+          market_pairs = []
+          response['data']['SupportPairs'].each do |pair|
+            base, target = pair.split('_')
```

Comment:
`https://api.c2cx.com/v1/ticker/?symbol=BTC_ETH` returns

```
{
    "code": 200,
    "message": "success",
    "data": {
        "timestamp": "1532052144",
        "high": 0.0651,
        "last": 0.0622,
        "low": 0.0618,
        "buy": 0.0622,
        "sell": 0.0624,
        "volume": 339.0794601449
    }
}
```

the base, target is inverted so,
`target, base` = pair.split('_')